environment-dev.yml

```diff
@@ -28,3 +28,5 @@ dependencies:
   - pytest-cov
   - python<=3.8
   - scipy
+  - pip:
```

Comment:
Just curious, is there any reason we use `pip` to install this package? From what I have been able to find, [`pre-commit` appears to be available as a `noarch` build on conda-forge.](https://anaconda.org/conda-forge/pre-commit)

Reply:
yeah, my environment wasn't resolving--I've fixed the issue now but it involved updating some pins